mcp/src/apis/getUsers.ts

```diff
@@ -5,13 +5,22 @@ import { ApiFactory } from '../shared/boilerplate/src/types.js';
 const inputSchema = {
   includeTimezone: z
     .boolean()
+    .default(true)
     .describe(
-      'If true, includes the timezones for each user. Not needed for most cases.',
+      'If true, includes the timezones for each user. Defaults to true.',
     ),
+  user_id: z
+    .string()
+    .optional()
+    .describe('The Slack user id (e.g. U0736TW20) of a single user to retrieve.'),
+  email: z
+    .string()
+    .optional()
+    .describe('The email address of a single user to retrieve.'),
   keyword: z
     .string()
     .min(0)
-    .nullable()
+    .optional()
     .describe(
       'Keyword to use to find partial matches on users. Will return users whose id (e.g. U0736TW20), name, real_name_normalized, or display_name_normalized contain the given keyword. This is case insensitive.',
```

Comment:
I think this is no longer accurate (given the changes here), and should be updated.